packages/core/src/pages/[...slug].tsx

```diff
@@ -212,6 +121,7 @@ export const getStaticProps: GetStaticProps<
       ...data,
```

Comment:
Not related to this line specifically but about the `throw errors[0]` a couple of lines before.

Should we log those errors?

```
if (errors.length > 0) {
  console.error(...errors)
  throw error[0]
}
```

Reply:
This way we could have this output show up on our logging system and can use that to debug more effectively.